superchain/configs/sepolia/superchain.yaml

```diff
@@ -7,7 +7,7 @@ l1:
 protocol_versions_addr: "0x79ADD5713B383DAa0a138d3C4780C7A1804a8090"
 superchain_config_addr: "0xC2Be75506d5724086DEB7245bd260Cc9753911Be"
 
-canyon_time: 1699981200  # Tue 14 Nov 2023 17:00:00 UTC
-delta_time: 1703203200   # Fri 22 Dec 2023 00:00:00 UTC
+canyon_time:  1699981200 # Tue 14 Nov 2023 17:00:00 UTC
+delta_time:   1703203200 # Fri 22 Dec 2023 00:00:00 UTC
 ecotone_time: 1708534800 # Wed 21 Feb 2024 17:00:00 UTC
-
+fjord_time:   1716998400 # Wed May 29 16:00:00 UTC 2024
```

Comment:
The spacing and time format (in the comment) diffs stand out here. 

I think the time format for Fjord should match the format of the other lines
```
date -u -r 1716998400
Wed 29 May 2024 16:00:00 UTC
```

Whitespace changes I am fine with, but we might want to file an issue to see if we can do the same alignment across all configs in this repo.